input/pagecontent/general-guidance.md

```diff
@@ -260,9 +260,9 @@ Example: Condition resource with coded condition, coded body site, laterality as
 
 ### Read/Search Syntax
 
-Searching resources is defined by the [FHIR RESTful API](https://hl7.org/fhir/R4/http.html) and included here for informative purposes. The [AU Core CapabilityStatements](capability-statements.html) document the server and client rules for the RESTful interactions described in this guide.
+Searching resources is defined by the [FHIR RESTful API](https://hl7.org/fhir/R4/http.html) and included here for informative purposes. The [AU Core CapabilityStatements](capability-statements.html) document the server and client rules for the RESTful interactions described in this guide. Examples in AU Core do not demonstrate the url encoding [rules for special characters](https://hl7.org/fhir/R4/search.html#escaping) e.g. "\|".
```

Comment:
The note feels a bit out of place here. Could you pls work on a few alternatives & maybe share them with @brettesler-ext and me at this stage?

Reply:
Moved to end of Read/Search Syntax section as per discussion with @dbojicic and @brettesler-ext. Thanks.